pkg/networkmanager/interfaces.js

```diff
@@ -1445,6 +1445,18 @@ PageNetworking.prototype = {
         $("#networking-add-bridge").click($.proxy(this, "add_bridge"));
         $("#networking-add-vlan").click($.proxy(this, "add_vlan"));
 
+        // HACK - hide "Add Team" if it doesn't work due to missing
```

Comment:
Why a line break for "bits"? Also, this could be a proper multiline comment using `/* */`.
I suggest:
```
        /* HACK - hide "Add Team" if it doesn't work due to missing bits
         * https://bugzilla.redhat.com/show_bug.cgi?id=1375967
         */
```